core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -80,7 +81,7 @@ class ReplicaFetcherThread(name: String,
 
   // Visible for testing
   private[server] val listOffsetRequestVersion: Short =
-    if (brokerConfig.interBrokerProtocolVersion >= KAFKA_2_1_IV1) 4
+    if (brokerConfig.interBrokerProtocolVersion >= KAFKA_2_1_IV1) 5
```

Comment:
Hmm.. We probably want to add a `KAFKA_2_2_IV0`. This change won't be backported to 2.1.